web/src/hooks/queries/useDisputeTemplate.ts

```diff
@@ -40,10 +45,29 @@ export const useDisputeTemplate = (disputeID?: string, arbitrableAddress?: `0x${
             true
           );
           console.log("useDisputeTemplate:", disputeTemplate);
+          const disputeTemplateInput = disputeTemplate?.templateData;
+          const dataMappingsInput = disputeTemplate?.templateDataMappings;
 
-          return disputeTemplate;
+          configureSDK({ apiKey: alchemyApiKey });
+
+          const initialContext = {
+            disputeID: disputeID,
+            arbitrable: arbitrableAddress,
+          };
+
+          console.log("dataMappingsInput", dataMappingsInput);
+          let data = {};
+          if (dataMappingsInput) {
+            const parsedMappings = JSON.parse(dataMappingsInput);
+            console.log("parsedMappings", parsedMappings);
+            data = await executeActions(parsedMappings, initialContext);
+          }
+          console.log("data", data);
+          const disputeDetailes = populateTemplate(disputeTemplateInput, data);
+
+          return disputeDetailes;
```

Comment:
```suggestion
          const parsedDataMappings = JSON.parse(dataMappings ?? "[]");
          const data = await executeActions(parsedDataMappings, initialContext);
          const disputeDetails = populateTemplate(templateData, data);
          return disputeDetails;
```

Reply:
Tried a different approach to your suggestion,
 ```
const data = dataMappings ? await executeActions(JSON.parse(dataMappings), initialContext) : {};
const disputeDetailes = populateTemplate(templateData, data);
```